src/main/java/com/t13g2/forum/storage/forum/ForumBookStorage.java

```diff
@@ -0,0 +1,145 @@
+package com.t13g2.forum.storage.forum;
+
+/**
+ *
+ */
```

Comment:
A lot of these empty header comments (add just to satisfy checkstyle right 😆), don't forget to fill them in later on!

Reply:
Sure!